draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -819,7 +848,8 @@ stopped by a police officer for exceeding a speed limit, Holder Binding may be n
 driving the car and presenting the license is the actual Holder of the
 license. The Verifier (e.g., the software used by the police officer)
 will ensure that a Holder Binding JWT is present and signed with the Holder's private
-key.
+key. Claims-based Holder Binding may be used as well, e.g., by including a
+picture of the user.
```

Comment:
related to an above comment, picture of the user would be a "biometric-based holder binding" in my mind.

```suggestion
key. Claims-based Holder Binding may be used as well, e.g., by including a
first name, last name and a date of birth that matches that of an insurance policy paper.
```